extensions/amp-story/1.0/amp-story.js

```diff
@@ -827,6 +827,16 @@ export class AmpStory extends AMP.BaseElement {
           accessService.areFirstAuthorizationsCompleted();
       accessService.onApplyAuthorizations(
           () => this.onAccessApplyAuthorizations_());
+
+      const firstPage = this.element.querySelector('amp-story-page');
```

Comment:
It should be faster to get `this.pages_[0].element`, since we can avoid querying the DOM.